docs/src/usage.md

```diff
@@ -430,20 +502,43 @@ Configure which input types Regula should search for:
 regula init --input-type tf --input-type cfn
 ```
 
+When using Regula with [Fugue](https://www.fugue.co), set the environment ID for Fugue to scan:
+
+```
+regula init --environment-id a29aec17-ab48-42dc-ade5-c0ba8b650c4c
+```
+
+Sync the built-in and custom rules from your [Fugue](https://www.fugue.co) tenant when running Regula locally:
+
+```
+regula init --environment-id a29aec17-ab48-42dc-ade5-c0ba8b650c4c --sync
+```
+
+!!! note
+    If you're using `--sync` in your configuration file, note that the `--sync` flag takes precedence over options that modify the rule set (`--exclude`, `--include`, `--only`, `--no-built-ins`). Those options will be ignored. To use those options with `regula run`, set `--sync=false`:
```

Comment:
```suggestion
    If you're using `--sync` in your configuration file, note that the `--sync` flag takes precedence over options that modify the rule set (`--exclude`, `--include`, `--only`, `--no-built-ins`). Those options will be ignored. To override the `--sync` in your configuration file, include `--sync=false` in your `regula run` command:
```

Reply:
Feel free to ignore or reword if that just makes it more confusing.

Reply:
Great suggestion, thanks!